remake.yml

```diff
@@ -32,18 +33,40 @@ targets:
   output:
     command: produceAnalysesOutputs(metRates)
 
-  tables:
-    command: makeTables(dataTable1=metRates, pathToDataTable2='output/RDatafiles/analyses.RData')
-  
+  output/data/table1.csv:
+    command: makeTable1(metRates, target_name)
+
+  output/data/table2.csv:
+    command: makeTable2(output, target_name)
+
   numbersList:
-    command: extractNumbersList(metRates, 'output/RDatafiles/analyses.RData')
+    command: extractNumbersList(metRates, output)
     
   output/figures/fig1.pdf:
     command: makeFigure1()
 
   output/figures/fig2.pdf:
-    command: makeFigure2(metRates, 'output/RDatafiles/analyses.RData')
+    command: makeFigure2(metRates, output)
 
   text/MS.md:
     depends: numbersList
     knitr: TRUE
+
+  output/MS.docx:
+    command: pandoc_convert("text/MS.md", output=target_name, wd=I("."), options = I(c("--bibliography", "text/references.bib", "--csl", "text/ecology.csl")))
```

Comment:
Note the wd=I(".") argument is needed to stop `pandoc_convert` from trying to change directories. Also, we can write the output file into `output` folder